wad.md

```diff
@@ -98,7 +99,21 @@ The following *compression types* are known:
 Paths of files in WAD archives are hashed using 64-bit
 [xxHash](http://cyan4973.github.io/xxHash/) with seed 0.
 Since paths are not stored in the archive in clear, they have to be guessed.
+Paths can also be restored from BIN files in the archive. This is a very good technique and will give you atleast 70% of the unknown hashes.
```

Comment:
`atleast` -> `at least`

Note: I'll probably make another section with common path formats, tips for retrieving them, etc.

Reply:
done